### ISSUE TITLE
Deprecated error shown on commenting on media

### ISSUE THREAD

**Issue by pooja-muchandikar** [rtCamp/rtMedia#1683]
```
Deprecated: get_option was called with an argument that is deprecated since version 5.5.0! The "blacklist_keys" option key has been renamed to "disallowed_keys". in /var/www/htdocs/wp-includes/functions.php on line 5057

Deprecated: class-phpmailer.php is deprecated since version 5.5.0! Use wp-includes/PHPMailer/PHPMailer.php instead. The PHPMailer class has been moved to wp-includes/PHPMailer subdirectory and now uses the PHPMailer\PHPMailer namespace. in /var/www/htdocs/wp-includes/functions.php on line 4963

Deprecated: class-smtp.php is deprecated since version 5.5.0! Use wp-includes/PHPMailer/SMTP.php instead. The SMTP class has been moved to the wp-includes/PHPMailer subdirectory and now uses the PHPMailer\PHPMailer namespace. in /var/www/htdocs/wp-includes/functions.php on line 4963
```

The above deprecated error is shown on commenting on media in lightbox. 

```
Test Environment
WordPress: v5.5. Beta RC 2
rtMedia: 4.6.4
```

**Comment by SID177** [comment]
@pooja-muchandikar 
These are BuddyPress' errors. When some other user comments on normal activity, I get same errors as above. I checked the rtMedia's code, its calling BuddyPress' function only.

**Comment by pooja-muchandikar** [comment]
The above errors mentioned in issue description are from BuddyPress. https://buddypress.trac.wordpress.org/ticket/8338 And is fixed in BuddyPress latest version 6.2.0. 

Cross-checked the issue with latest BP versiob 6.2.0 and the error is fixed. Since the issue if fixed closing this issue.
